.pre-commit-config.yaml

```diff
@@ -4,7 +4,7 @@ repos:
     hooks:
       #- id: check-yaml
       #  args: [--allow-multiple-documents]
-      - id: check-json
```

Comment:
Had a false positive that was blocking me:
```
check json...............................................................Failed
- hook id: check-json
- exit code: 1

components/ws-manager-api/typescript/.vscode/launch.json: Failed to json decode (Expecting property name enclosed in double quotes: line 18 column 9 (char 543))
```

Reply:
If that's the case, exclude that single file, not the whole check. This is akin to saying we have a failing test, I'll just comment it out.